csm/core/controllers/usl.py

```diff
@@ -85,8 +85,8 @@ def _validate_api_key(self) -> None:
 
 
 @Decorators.decorate_if(not Options.debug, _Proxy.on_loopback_only)
-@CsmView._app_routes.view("/usl/v1/saas")
-@CsmView._app_routes.view("/usl/v2/saas")
+@CsmView._app_routes.view("/api/v1/saas")
+@CsmView._app_routes.view("/api/v2/udx_saas")
```

Comment:
api should still be "usl" we should not change it

Reply:
okay .. will update this @kupranay

Reply:
As a matter of fact, this is not part of the USL API.  If we want to update it with the `/api` prefix, that would be even better.

Reply:
@tb-s  you are right. we should update it, but we will take it as a separate task